### ISSUE TITLE
iPad scrolling

### ISSUE THREAD

**Issue by Prokophs** [htdt/parallax#1]
First of all your work is f**n amazing and well done.
I've tried both the plugin and your website on an iPad and scrolling is somehow a trouble.
Once you have made a scroll you have to wait for the scroll to finish before you can perform another one.

By the way, is there any chance to get this for vertical scrolling instead of horizontal?

**Comment by juancpgo** [comment]
Yeah, awesome work, but the scrolling is not very user friendly at the iPad...
